src/app-layout/__integ__/runtime-drawers-controllability.test.ts

```diff
@@ -7,7 +7,7 @@ import createWrapper from '../../../lib/components/test-utils/selectors';
 const wrapper = createWrapper().findAppLayout();
 
 for (const visualRefresh of [true, false]) {
-  for (const pageName of ['runtime-drawers', 'runtime-drawers-imperative']) {
+  for (const pageName of ['runtime-drawers', 'runtime-drawers-imperative', 'runtime-drawers-with-updates']) {
```

Comment:
These two pages represented different tools integration use-cases. The new "with-updates" page does not represent a new use-case, so it is not needed here

Reply:
updated